### PR TITLE
Create site

### PR DESCRIPTION
**Pull request by eternoendless**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description  | Added a new site for the project

- Blog sends you to Build
- Contribute to the appropriate section on Devdocs
- Resources contains a link to the code of conduct and the user groups part

TO DO:

- [ ] Slack section
- [ ] Find links for the page footer
- [ ] Complete the About section (hidden for now)
- [ ] Move the theme to the org (in my fork right now)

For later:

- Move the maintainer's guide from the devdocs

Screenshot:

Screencap

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "themes/ps-org-theme"]
+	path = themes/ps-org-theme
+	url = https://github.com/eternoendless/ps-org-theme.git
```
Comment (Progi1984) [inline .gitmodules:3]: Please move it to PrestaShop organisation

File: content/about.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+title: "About the PrestaShop Project"
+---
+
+## What is PrestaShop?
+
+PrestaShop is an open-source e-Commerce content management system
```
Comment (Progi1984) [inline content/about.md:7]:
```suggestion
PrestaShop is an open-source e-Commerce content management system.
```

**Review by Progi1984 — COMMENTED** [review]
Some feedback

File: .gitmodules (hunk 1 above)
Comment (eternoendless) [reply .gitmodules]: Done

**Review by Progi1984 — APPROVED** [review]

File: config/_default/params.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    black: "#000"
+    white: "#fff"
+    light: "#E5E1F9"
+
+pluralizeListTitles: false
+
+projectUrls:
+    gitHub: "https://github.com/PrestaShop/PrestaShop/"
+    reportBug: "https://github.com/PrestaShop/PrestaShop/issues"
+    slack: "https://github.com/PrestaShop/open-source/blob/master/slack/readme.md"
```
Comment (matks) [inline config/_default/params.yml:24]: I think the file `/slack/readme.md` will now be available as one webpage on prestashop-project.org ? One resources page?

**Review by matks — APPROVED** [review]
Code is ✅  for me. I added a suggestion.

**Comment by ttoine** [comment]
How can I add the missing sections?
Do we merge that first and then I'll do pull requests?

**Review by PierreRambaud — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]
lgtm

**Comment by ttoine** [comment]
so, now that it is merged, is it online somewhere already ?

**Comment by Progi1984** [comment]
Nope. Why ?
- The DNS rule has not been done
- @eternoendless has detected an error in Github Actions :wink:

**Comment by ttoine** [comment]
because I don't know, so I ask ;-)
can't wait to see it live and be able to add content to it, btw.

**Comment by Progi1984** [comment]
The why is for the explanation.

**Pull request merged**